### PR TITLE
Adds TLS support for STOMP connections

### PR DESCRIPTION
**Pull request by kazeborja**
This PR aims to provide TLS support for STOMP connections using certificates.
This is needed as at CERN TLS authentication is handled using certificates.

### CODE REVIEW

File: config/config.yaml [hunk 1]
```diff
@@ -1,9 +1,22 @@
+# Select which protocol to use in order to connect to the MQ
+mq: amqp/stomp
```
Comment (djw8605) [inline config/config.yaml:2]: You should comment out this line, otherwise it fails testing.
```suggestion
# mq: amqp/stomp
```

**Review by djw8605 — CHANGES_REQUESTED** [review]
The Go standard is to use tabs, not spaces, for indention.  The rest of the files are tab separated.  I believe you can reformat the files with the command:
```
$ go fmt
```

**Comment by kazeborja** [comment]
Thanks Derek for the pointers, I always forget about go fmt tool

**Review by djw8605 — APPROVED** [review]
LGTM

**Pull request merged**